features/comments_and_kudos/kudos.feature

```diff
@@ -125,3 +125,93 @@ Feature: Leave kudos
       And the email should contain "guest"
       And the email should contain "Awesome Story"
       And the email should contain "Another Awesome Story"
+
+  Scenario: Leave kudos on an admin post
+    Given I have posted an admin post
+
+    When I am logged out as an admin
```

Comment:
I'd make this an `And` and remove the blank line, but it looks as if this step shouldn't be necessary anyway. Following through the complex chain of variations on `I am logged...` steps, the bottom of the pile is a step definition which calls `I am logged out` which in turn destroys any admin user sessions, so this line can probably be removed. (Same applies to subsequent tests following the same format)